app/retail/emails.py

```diff
@@ -297,6 +297,22 @@ def render_bounty_startwork_expired(to_email, bounty, interest, time_delta_days)
     return response_html, response_txt
 
 
+def render_gdpr_update(to_email):
+
+    params = {
+        'subscriber': get_or_save_email_subscriber(to_email, 'internal'),
+        'terms_of_use_link': '',
```

Comment:
Should these params have value to reverses of their relevant urls?

Reply:
Yes. I was about to comment. But all the links are still not active in the master branch. Hence I will add them once the other PRs get merged.